lib/chef-dk/command/verify.rb

```diff
@@ -452,6 +451,71 @@ def foo
         end
       end
 
+      unless Chef::Platform.windows?
+        add_component "delivery-cli" do |c|
+          # We'll want to come back and revisit getting unit tests added -
+          # currently running the tests depends on cargo , which is not included
+          # in our package.
+          c.base_dir = "bin"
+          c.smoke_test do
+            tmpdir do |cwd|
+              sh!("delivery setup --user=shipit --server=delivery.shipit.io --ent=chef --org=squirrels", cwd: cwd)
+            end
+          end
+        end
+
+        add_component "git" do |c|
+          c.base_dir = "embedded/bin"
+          c.smoke_test do
+            sh!("#{bin("git")} config -l")
+          end
+          c.integration_test do
```

Comment:
If so we should definitely consider running more than the `--unit` group in our testing gate:
https://github.com/chef/chef-dk/blob/master/ci/verify-chefdk.sh#L45

It worries me we don't actual exercise real functionality of `delivery` and `git` anywhere in our CD pipeline.